testsuite/CHANGELOG

```diff
@@ -17,6 +17,7 @@ and https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests
          tyler.je.reddy
 
   * 0.16
+    - Added unit tests for MDAnalysis.analysis.psa exception handling
```

Comment:
@richardjgowers, I (and others such as @jbarnoud ) had indeed asked @tylerjereddy to add entries to the tests CHANGELOG. 

I think we need to clear up among ourselves what should go in there in order to avoid the friction exhibited in this comment thread.